doc/add-data-new-version.md

```diff
@@ -4,46 +4,44 @@ For bedrock edition see [bedrock.md](bedrock.md)
 
 | file name | auto? | how to get it | notes |
 | -- | -- | -- | -- |
-| protocol.json | No | Follow [Pre-release Protocol][3] |
-| blocks.json | Yes | Use [Burger][1], then use [burger-extractor][2] |
-| items.json | Yes | Use [Burger][1], then use [burger-extractor][2], then use [turbo-invention][6] |
-| entities.json | Yes | Use [mcdata-generator][20] and run `extractPcEntityMetadata.js` script in tools/js to generate entity metadata in entities.json and protocol.json |
-| recipes.json | Yes | Use [Burger][1], then use [burger-extractor][2] | should eventually be changed to native data generators |
-| blockCollisionShapes.json | Yes | Use [McDataExtracting][4] |
-| commands.json | No? |Use [mc-data-command-generator][5] | link to jar files have to be manually added |
-| biomes.json | Yes | Use [Burger][1], then use [burger-extractor][2] | [extra info][13] |
-| instruments | No | [wiki.vg block actions][11] |
-| materials.json | No | [import from previous version](18) or [fix it properly](19) |
-| windows.json | No | Look at [wiki.vg inventories][7] |
-| version.json | No | Look at [Protocol Versions][9] | [wiki.vg protocol numbers][8] |
-| effects.json | No | Look at [minecraft wiki effects][12] |
-| enchantments.json | No | Duplicate enchantments.json from the latest version and add manually the missing enchantments. Enchantments data could be found by looking into the deobfuscated classe files of each enchantments, as well as in the Enchantments registry. |
-| language.json | Yes | Use [minecraft-jar-extractor][10] |
-| particles.json | Yes | Use [Burger][1], then use [burger-extractor][2] |
-| blockLoot.json | Yes | Use [minecraft-jar-extractor][10] |
-| entityLoot.json | Yes | Use [minecraft-jar-extractor][10] |
-| mapIcons.json | No | Icons can be found in the Minecraft jar file where they are added as a single sprite. The file location is `/assets/minecraft/textures/map/map_icons.png`. Alternatively you might be able to look up the icons from the following page on the [Minecraft wiki][15] or from [wiki.vg][16]. | [minecraft-data pr mapIcons][14] |
```

Comment:
Why remove the file path for this, is it incorrect?

Reply:
Its a correct file path but not useful. Since it doesn't label the icons at all, and doesn't provide the information needed to fill out the mapIcons.json file.

Reply:
For example https://github.com/extremeheat/extracted_minecraft_data/blob/client1.20.4/client/net/minecraft/world/level/saveddata/maps/MapDecoration.java  vs the map image file. https://github.com/PrismarineJS/minecraft-assets/blob/master/data/1.20.2/map/map_icons.png